source/jsonnet/england-wales/household/blocks/individual/individual_introduction.jsonnet

```diff
@@ -4,6 +4,7 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
 {
   type: 'Interstitial',
   id: 'individual-introduction',
+  page_title: 'Introduction to individual questions',
```

Comment:
Given this is not visitor, the proxy page is missing the correct title.

Reply:
Have updated these now